docs/openapi.yaml

```diff
@@ -78,157 +78,163 @@ paths:
       parameters:
       - in: query
         name: category
-        description: "Term to search for in resource."
+        description: "Result category code, category codes are defined in the Cristin API documentation."
         required: false
         schema:
           type: string
       - in: query
         name: contributor
-        description: "Term to search for in resource."
+        description: "Author's name or Cristin person id."
         required: false
         schema:
           type: string
       - in: query
         name: created_before
-        description: "Term to search for in resource."
+        description: "Results created before and inclusive the given date, (yyyy-mm-dd), e.g: 2005-03-17"
         required: false
         schema:
           type: string
       - in: query
         name: created_since
-        description: "Term to search for in resource."
+        description: "Results created since and inclusive the given date, (yyyy-mm-dd), e.g: 2005-03-17"
         required: false
         schema:
           type: string
       - in: query
         name: doi
-        description: "Term to search for in resource."
+        description: "DOI name (e.g. doi=10.1000/123456)"
         required: false
         schema:
           type: string
       - in: query
         name: fields
-        description: "Term to search for in resource."
+        description: "'fields=all' gives a list of result objects with all available fields. If this parameter is omitted, a list of result summaries with fewer fields will be returned."
```

Comment:
what other values do we support? We use schema type 'string' underneath but here it seems like a enum or boolean? We should document all posibilities if its a enum